src/pages/signin/PasswordForm.js

```diff
@@ -137,6 +137,7 @@ class PasswordForm extends React.Component {
                         onChangeText={text => this.setState({password: text})}
                         onSubmitEditing={this.validateAndSubmitForm}
                         blurOnSubmit={false}
+                        autoCapitalize="none"
```

Comment:
This is needed for?

Reply:
If user keyboard settings with Auto-Capitalization on, when tap on the eye to toggle secure text visible, the keyboard will change from lower case (if it's lower case mode when user tap the last character) to upper case mode, set `autoCapitalize` to `none` can avoid that, the keyboard will keep and remain in lower case mode.

Reply:
Pardon my ignorance, but I didn't get what impact does it have on our PR. Can you please help me with this?

Reply:
> If user keyboard settings with Auto-Capitalization on, when tap on the eye to toggle secure text visible, the keyboard will change from lower case (if it's lower case mode when user tap the last character) to upper case mode, set `autoCapitalize` to `none` can avoid that, the keyboard will keep and remain in lower case mode.

I found this issue when I test https://github.com/Expensify/App/issues/9444, and I think we can fix it in our current PR. Maybe I post it as a new issue and fix it later is a better idea, we should remove `autoCapitalize="none"`. What's your opinion?